scripts/apps/authoring/authoring/article-url-fields.tsx

```diff
@@ -31,18 +32,17 @@ export class ArticleUrlFields extends React.Component<IProps, IState> {
     }
     addUrl() {
         this.setState({
-            urls: this.state.urls.concat('https://'),
+            urls: this.state.urls.concat({url: 'https://', description: ''}),
         }, () => {
             this.props.onChange(this.props.fieldId, this.state.urls);
         });
     }
-    handleChange(index, event) {
+    handleChange(index, key, event) {
```

Comment:
`field` instead of `key` would sound clearer to me. You could also add a typing `field: keyof IUrl`